### ISSUE TITLE
Swupd Error using latest installer

### ISSUE THREAD

**Issue by dedguy21** [clearlinux/clr-installer#532]
**Describe the bug**
Swupd fails to download packages during install after using Advanced Disk Configuration where SDA1 is /boot SDA2 is /  SDB2 is swap and SDB1 is /home (CLR_HOME)

**To Reproduce**
Have two disk on computer
1. Go to 'Advance  Disk Configuration Option.' use cgdisk to format disks make sure labled to Clear Linux specs ie CLR_BOOT, CLR_ROOT, CLR_SWAP, CLR_HOME. Also make sure to use correct hex codes for each partition ie swap = 8200  Linux-home = 8304 etc
2. Click on 'write and then quit'
3. Additional Bundles select any additional bundle 
Scroll down to install and confirm you want to install. The process will run formatdisk and fs process, it will get to the point where it is ready to download additional packages and then...

![IMG_20190918_210445](https://user-images.githubusercontent.com/49103740/65262940-680dbd00-dad9-11e9-951c-907e1939dcdc.jpg)
![IMG_20190918_210644](https://user-images.githubusercontent.com/49103740/65262955-6e039e00-dad9-11e9-86e6-4d8b1053458d.jpg)

**Expected behavior**
I expect that swupd install bundles into machine or advise if disk format failed (I didn't get a disk configuration error though so it might not be an issue)

**Screenshots**

**Environment (please complete the following information):**
 - Clear Linux OS Version: [sorry don't know off hand but the latest version as of 9-18-19]
 - Installer Version [2.2-v]
 - Platform: 
Thinkpad 440p
Seagate SSD 
Western Digital HDD
15 gigs ram
Haswell i5 3.0ghz 

**Additional context**
When I didn't select additional bundles and didn't use advance disk configuration tool it installed without issue. Which is why I closed a previous bug report. But when I tried to add bundles I got the screen shot errors.

**Comment by mdhorn** [comment]
@dedguy21 Thanks for the report. Any chance of getting the files list on the first screen as attachments?
```
/root/pre-install-clr-installer.yaml
/root/clr-installer.log
```

One thing I noted from your comments is using `CLR_HOME` which is not valid syntax.
Only the ROOT, SWAP, AND BOOT have unique labels -- these are the minimum requirements. All other partitions are identified with `CLR_MNT_/<mnt point>`.
Also, we recommend just selecting the filesystem system (ext4 of 8300 for example), and let the install handle setting unique codes for when known partitions like root or home.

We have an example document here:
https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-server.html#advanced-installation

The only error I can see from the screenshot implies the problem happened while making the user account -- I assume you created a user during setup? Was it an admin user?

**Comment by dedguy21** [comment]
I'll will provide log and y'all if I get another error tonight.

To answer previous question: I was create myself as a user with admin
privileges which obviously has never caused issue before. I'm guessing you
have something to do with trying to creep the home folder for the user if
my syntax is wrong.

**Comment by mdhorn** [comment]
@dedguy21 Were you able to get this resolved?
We also have a new release of the clr-installer with several small bug fixes in 2.3.0 which released in Clear Linux OS 31150.

**Comment by dedguy21** [comment]
Thanks. I just installed in one disk and moved /home and /var after
install.

I'll download the new installer and see if it allows for two disk install.

Thanks
